src/Appwrite/Statsd/Statsd.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Appwrite\Statsd;
+
+use Utopia\App;
+
+class Statsd
```

Comment:
I think a better name would be `Stats` as this is an Appwrite specific wrapper. Would also allow to decouple from StatsD easily in the future.